src/lib/es2023.array.d.ts

```diff
@@ -183,15 +183,15 @@ interface Int8Array {
     ): number;
 
     /**
-     * Copies the array and returns the copy with the elements in reverse order.
+     * Returns a copy of an array with its elements reversed.
      */
     toReversed(): Uint8Array;
 
     /**
-     * Copies and sorts the array.
+     * Returns a copy of an array with its elements sorted.

```

Comment:
I think it’s important that all of these use the language “the array” since these are instance methods; it’s not just any array, it’s `this` array.

```suggestion
     * Returns a sorted copy of the array.
```